src/pages/journal/journal.ts

```diff
@@ -18,11 +11,8 @@ import { AuthenticationProvider } from '../../providers/authentication/authentic
 import * as journalActions from './journal.actions';
 import { StoreModel } from '../../common/store.model';
 import {
-  getError,
-  getIsLoading,
-  getLastRefreshed,
-  getLastRefreshedTime,
-  getSlotsOnSelectedDate,
+  getError, getIsLoading, getLastRefreshed,
```

Comment:
Are we planning on allowing multi+multi imports? Is there a limit of imports per line?

Reply:
Yes. No, apart from the 120 line length limit